lib/src/view/menu.dart

```diff
@@ -180,6 +182,16 @@ really want to exit without saving.'''
       (MenuDropdownItem()
         ..on_click = ((_) => props.dispatch(actions.ExportCodenanoFile()))
         ..display = 'Export codenano')(),
+      DropdownDivider({}),
+      (MenuBoolean()
+        ..value = props.save_dna_in_local_storage
+        ..display = 'Save DNA Design in localStorage'
+        ..tooltip = '''\
+Saves DNADesign in localStorage on every edit. Disabling this minimizes the time needed to render large strands.'''
```

Comment:
In the tooltip and display, just call it "design", not "DNADesign". The phrase "DNADesign" might not be familiar to the user unless they use the Python scripting interface.

Also, change the phrase "large strands" to "large designs". (A design can be large because it has many small strands.)